packages/components/src/Autocomplete/Autocomplete.stories.mdx

```diff
@@ -0,0 +1,343 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Button } from "@jobber/components/Button";
+import { Autocomplete } from "@jobber/components/Autocomplete";
+import { useState } from "react";
+import cloneDeep from "lodash/cloneDeep";
+
+<Meta title="Components/Autocomplete" component={Autocomplete} />
+
+# Autocomplete
+
+An autocomplete allows a user to quickly pick a preset value from a larger list
+of possible options.
+
+If the number of available options is smaller consider a [Select](select)
+instead.
+
+```ts
+import { Autocomplete } from "@jobber/components/Autocomplete";
+```
+
+<Canvas>
+  <Story name="Autocomplete">
+    {() => {
+      const options = [
+        { value: 1, label: "Nostromo" },
+        { value: 2, label: "Rodger Young" },
+        { value: 3, label: "Serenity" },
+        { value: 4, label: "Sleeper Service" },
+        { value: 5, label: "Enterprise" },
+        { value: 6, label: "Enterprise-D" },
+      ];
+      const [value, setValue] = useState();
+      return (
+        <Autocomplete
+          value={value}
+          initialOptions={[]}
+          onChange={setValue}
+          getOptions={getOptions}
+          placeholder="Search for something"
+        />
+      );
+      function getOptions(text) {
+        if (text === "") {
+          return options;
+        }
+        const filterRegex = new RegExp(text, "i");
+        return options.filter(option => option.label.match(filterRegex));
+      }
+    }}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable of={Autocomplete} story="Autocomplete" />
+
+---
+
+## With details
+
+You can add a description to an option, which will appear below the option's
+label. You can also add a detail to an option which shows on the option's the
+bottom right. Both of these are optional per option.
+
+<Canvas>
+  <Story name="With details">
+    {() => {
+      const options = [
+        {
+          value: 1,
+          label: "Sulaco",
+          description: "They mostly come at night, mostly.",
+          details: "LV-426",
+        },
+        { value: 2, label: "Nostromo", details: "LV-426" },
+        { value: 3, label: "Serenity", description: "I aim to misbehave." },
+        { value: 4, label: "Sleeper Service" },
+        { value: 5, label: "Enterprise" },
+        {
+          value: 6,
+          label: "Enterprise-D",
+          description: "Tea, earl grey, hot.",
+          details: "NCC-1701D",
+        },
+      ];
+      const [value, setValue] = useState();
+      return (
+        <Autocomplete
+          value={value}
+          initialOptions={options}
+          onChange={setValue}
+          getOptions={getOptions}
+          placeholder="Search for something"
```

Comment:
let's move this to `args` except for `value` and `onChange`

Reply:
I think `initialOptions` and `getOptions` should be outside the `args` too as they're getting these values from the constant, right?